### PR TITLE
[FIX] Search method should only search on fiscal month date range

### PR DESCRIPTION
**Pull request by benwillig**

### CODE REVIEW

**Review by adrienpeiffer — APPROVED** [review]
:+1:

File: account_move_fiscal_month/models/account_move.py [hunk 1]
```diff
@@ -12,8 +12,15 @@ class AccountMove(models.Model):
 
     date_range_fm_id = fields.Many2one(
         comodel_name='date.range', string="Fiscal month",
+        domain=lambda self: self._get_date_range_fm_domain(),
```
Comment (gurneyalex) [inline account_move_fiscal_month/models/account_move.py:15]: why the lambda?

Comment (benwillig) [reply account_move_fiscal_month/models/account_move.py:15]: The lambda is used to be able to inherit from ```_get_date_range_fm_domain```.

**Review by Cedric-Pigeon — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Why runbot fails?

**Review by rousseldenis — APPROVED** [review]
Code review. Makes sense.

@pedrobaeza All :green_apple:

**Review by pedrobaeza — APPROVED** [review]

**Comment by Cedric-Pigeon** [comment]
@pedrobaeza This one could be merged, isn't it?

**Pull request merged**